it-tools.yaml

```diff
@@ -0,0 +1,85 @@
+package:
+  name: it-tools
+  version: 2024.10.22
+  epoch: 0
+  description: Collection of handy online tools for developers, with great UX
+  copyright:
+    - license: GPL-3.0
+  options:
+    no-depends: true
+  dependencies:
+    runtime:
+      - ${{package.name}}-nginx-config
+      - nginx
+
+environment:
+  contents:
+    packages:
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - git
+      - nginx-package-config
+      - node-gyp
+      - nodejs
+      - npm
+      - pnpm
+  environment:
+    NPM_CONFIG_LOGLEVEL: "warn"
+
+pipeline:
+  - runs: |
+      # The version includes a commit from one of the pull requests that contribute to the new version
+      git clone --depth=1 https://github.com/CorentinTh/it-tools.git
+      cd it-tools
+      package_version="${{package.version}}"
+      git fetch --tags
+      latest_tag=$(git tag -l "v${package_version}-*" | sort -V | tail -n 1)
+
+      git checkout ${latest_tag}
```

Comment:
Should we create a new ReleaseMonitor and use git-checkout` pipeline here instead? That way we able to access latest tag.